course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -130,7 +136,54 @@ describe('Unit testing PeopleController', function() {
             $httpBackend.expectGET(courseInstanceURL)
                         .respond(200, JSON.stringify(ci));
             $httpBackend.flush(1);
-            expect(scope.title).toEqual(ci.title);
+            expect(scope.courseInstance['title']).toEqual(ci.title);
+            expect(scope.courseInstance['school']).toEqual(ci.course.school_id);
+        });
+    });
+
+    describe('getFormattedCourseInstance', function() {
+        var ci;
+        beforeEach(function() {
+
+            ci = {
+                course_instance_id: '1234',
+                title: 'Test Title',
+                sites: [
+                    {
+                        external_id: 'https://x.y.z/888',
+                        site_id: '888',
+                    }
+                ],
+                course :{
+                    school_id: 'ABC',
+                    registrar_code_display: '2222',
+                    course_id : '789',
+                },
+                term:{
+                    display_name: 'Summer 2015',
+                    academic_year : '2015',
+                },
+                primary_xlist_instances :[],
+            };
+        });
+
+        it('format the course instance data for the UI ', function() {
+            controller = $controller('PeopleController', {$scope: scope});
+            $httpBackend.expectGET(courseInstanceURL)
```

Comment:
We've already tested above that the ajax fetch of the course instance works.  This test gets a little simpler if we just set the course instance before instantiating the controller...we don't need to mock the ajax call, since it never happens.